docs/source/intervention_models/multiple_myeloma_treatment/index.rst

```diff
@@ -607,33 +607,115 @@ match those generated within Foundry. It requires access to all the files in J:\
+We then estimate the population-level probabilities by aggregating treatment assignment model predictions for this dataset.
+
+The key is that, at each postprocessing step, we not only apply the step to the actual simulant-level probabilities for the timestep, but also to the estimated population-level probabilities.
+This means that they are kept in sync: at each point in the postprocessing algorithm, the estimated population-level probabilities are our best estimate of what a large population's mean probabilities
+would be **just before that step**.
+
+The estimated probabilities before any postprocessing steps are applied, calculated for each line number at the start of each calendar year, can be found at
+J:\\Project\\simulation_science\\multiple_myeloma\\data\\treatment_model_input\\2022_08_10\\population_level_probabilities.csv.
+At each timestep, the probabilities used are a linear interpolation between the adjacent years according to the date at that timestep.
+
+China adjustments
+^^^^^^^^^^^^^^^^^
+
+Goals
+'''''
+
+Our China adjustment postprocessing rules are designed so that:
+
+* We approximately match our population-level NDMM induction coverage targets for each cytogenetic risk grouping. The target values are interpreted as proportions **among treatments except for Other and Other+ASCT** for lower-granularity groups of regimen categories: Dara quadruplets, Dara triplets, other triplets, doublets, and regimen categories not seen in China (always set to 0). Note that these induction targets are not stratified by ASCT.
+    * Dara quadruplets: Dara+PI+IMID+Dex and Dara+PI+Chemo+Dex, with and without ASCT
+    * Dara triplets: Dara+PI+Dex and Dara+IMID+Dex, with and without ASCT
+    * Other triplets: PI+IMID+Dex and Chemo+PI+Dex, with and without ASCT
+    * Doublets: PI+Dex and IMID+Dex, with and without ASCT
+    * Not seen in China (probabilities set to 0): Chemo+IMID+Dex, with and without ASCT 
+* We approximately match our population-level RRMM induction coverage targets, which are for the same groups of regimen categories, but not separated by cytogenetic risk.
+* We approximately match our population-level NDMM targets for the probability of treatment containing ASCT or not.
+* As much as possible, we do not change the probability ratios between simulants for each regimen category. In other words, if Simulant 1 was twice as likely as Simulant 2 to receive PI+IMID+Dex before these rules, that should still be the case after these steps. However, we are forced to compromise on this requirement when we normalize each simulant's probabilities to sum to 1.
+* Within any of the above groups of regimen categories, for a single ASCT value (e.g. Dara triplets with ASCT), we do not change the ratios of probabilities between the regimen categories within it. This is true not only for the population but also at the simulant level.
```

Comment:
So no correlation between ASCT probability and treatment category? This wasn't immediately obvious to me and seems like something we should flag!

Reply:
There is a correlation between ASCT and induction, informed by Flatiron (since our China coverage targets do not tell us about the joint distribution).

What I was trying to express with this bullet point is that the "sub-category" breakdowns are retained. From the perspective of our China targets, all Dara triplets with ASCT are the same. So how that group is split between Dara+PI+Dex+ASCT and Dara+IMID+Dex+ASCT is retained from our Flatiron-trained model.